QuasarHelios2015/src/org/team1540/quasarhelios/Elevator.java

```diff
@@ -29,20 +33,18 @@
         }
     }
 
-    private static final BooleanStatus raising = new BooleanStatus();
-    private static final BooleanStatus lowering = new BooleanStatus();
-    private static final BooleanStatus goingToMiddle = new BooleanStatus();
+    private static final BooleanStatus raising = new BooleanStatus(false);
+    private static final BooleanStatus lowering = new BooleanStatus(false);
```

Comment:
Note that the `false` is redundant. It's implied. If you think it's important, you can keep this, but consider whether or not you do.